src/slider.jsx

```diff
@@ -323,17 +314,84 @@ const Slider = React.createClass({
       document.addEventListener('touchcancel', this._dragTouchEndHandler, false);
     }
     this._onDragStart(e);
+
+    // Cancel scroll and context menu
+    e.preventDefault();
   },
 
   _onHandleMouseDown(e) {
     if (document) {
       document.addEventListener('mousemove', this._dragHandler, false);
       document.addEventListener('mouseup', this._dragEndHandler, false);
-      this._toggleSelection('none');
+
+      // Cancel text selection
+      e.preventDefault();
+
+      // Set focus manually since we called preventDefault()
+      ReactDOM.findDOMNode(this.refs.handle).focus();
```

Comment:
I know `ReactDOM` was already imported so this isn't your fault, but I'm thinking we could get away with `this.refs.handle.focus()` instead since the ref is assigned to a dom node. We should be able to remove the `ReactDOM` import after that. :grin: